typescript/web/src/components/website/Navbar/NavContent.tsx

```diff
@@ -22,7 +23,9 @@ import { links } from "./_data";
 
 const StarIcon = chakra(RiStarLine);
 
-const GitHubButton = ({ isMobile }: { isMobile?: boolean }) => {
+type GitHubButtonProps = ButtonProps & { isMobile?: boolean };
+
+const GitHubButton = ({ isMobile, ...props }: GitHubButtonProps) => {
```

Comment:
The button is not aligned with the other elements of the top bar, it is a bit shifted to the top

<img width="873" alt="Capture d’écran 2022-01-26 à 10 05 40" src="https://user-images.githubusercontent.com/43350109/151134243-a5d7a5e5-4bb4-4d06-8ce8-e33916ce177e.png">

Reply:
That's strange, IIRC it used to be... I'll fix it. Thanks for noticing this 👍